.travis.yml

```diff
@@ -13,11 +13,10 @@ addons:
 
 deploy:
   provider: npm
-  email: unindented@gmail.com
   api_key:
     secure: gA6QGEyXv6rOHHv9OKurFaPlhDjIbyxGF+45vamtkPlL/82pDErd/GK/oWWEP9MYdyjcp4QpRlLn/CTXSaFB6RaA8ceR2PwS7J02f5jZPOMDGOUD/apyoUKSaIEG1tVDegaoNCuMEKSG7p8zGuZEJf45Qwzfkm1rkQ7AicG9wzNrv5IMZzETJcdSAJyaVV3jm3FP+o6QdsOg1M9I+rqJaVsTc4tO5t+sUCMRCtZkrBmRQ+p2jbCl+gYXPSMCbw3dMso/gdqEcHCs1LWi5oj5Jlo94zP6i0zEEdZMagj7IDwvVAgbA/OTjhcEsKrNj4nJEa0JNLjvYdq2LZ6cQWFdWR8OVnYHbhqG9dXEeuVDkwQcnlp3+7FioZ7DCzybck6XZhBepXYY6h3J08jCf9lpiQBs/U/pij20f39wx1tTM3hzlfespX0qpejukak7p47ko050udNH7B1C64NvFi7q2Vg2h8jEZ4rYZ1qYmzcopTV54zhRzoyKIuH3+Z5x2cb85NSW5KSZnYXDf2o/SDColDz9se8uunf0PlkY2fZP+1t42QbyVI6JGZICiXdoX1CEUneGV+iiPDCEGVTm1LgaN6cORuLZz4LKIerxV6NRNIYODwaJVQ3q24vwndVeXqrNic1wJRw0B9F5t3ttRcPqsvW+XRa1wVxEqQUaNp9BoMk=
```

Comment:
This is probably not the right API key

Although I didn't know that Travis CI could deploy stuff! That is cool!